src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -5,12 +5,15 @@
 using NBitcoin;
 using NBitcoin.BouncyCastle.Math;
 using NBitcoin.Crypto;
-using NLog.Extensions.Logging;
 using Stratis.Bitcoin.Features.Consensus.CoinViews;
 using Stratis.Bitcoin.Utilities;
 
 namespace Stratis.Bitcoin.Features.Consensus
 {
+    /// <summary>
+    /// Provides functionality for checking validity of PoS blocks.
+    /// See <see cref="PosMinting"/> for more information about PoS solutions.
```

Comment:
hmm, damn, this reference is not recognized by MSVS because consensus does not depend on miner, not sure what to do with it, but probably nothing :/ 
maybe we could at least put a fully qualified name here `Stratis.Bitcoin.Features.Miner.PosMinting`, although it does not really help, but ...